contrib/opentimelineio_contrib/application_plugins/tests/test_rv_reader.py

```diff
@@ -196,11 +201,16 @@ def test_read_otio_file(self):
                 if not rvc.connected:
                     time.sleep(.5)
 
-                if attempts == 10:
+                if attempts == 20:
                     raise socket.error(
                         "Unable to connect to RV!"
                     )
 
+            # some time can pass between the RV connection
+            # and the complete startup of RV
+            print("Waiting for RV startup to complete")
+            time.sleep(10)
```

Comment:
Is this sleep(10) meant to be in addition to the sleep(.5) above? So after rvc is connected it still needs to wait for RV to finish launching? I wonder if there's a way to detect that RV is fully ready within the while loop that would avoid waiting longer than necessary.
I know this test is not usually run for most people, so it might not matter too much, but the faster the tests are the more likely they will be run regularly.
If this is not an easy fix, I'm fine with this PR as-is. Someday we can have CI runners with RV installed and this can be reassessed at that time.

Reply:
> Is this sleep(10) meant to be in addition to the sleep(.5) above?

Yes, those sleeps are waiting for rvNetwork to connect whereas this one is waiting between the time the connection succeeds and the full startup of RV. 

> So after rvc is connected it still needs to wait for RV to finish launching? 

Yes, there is a delay between the time the server loads and RV is fully initialized.  I am running a debug build of RV and on an 8 year old Mac, so those factors probably make the delay more significant on my machine.  I can try a release build if you think it's worth seeing if we can lower that value, but I thought i was worth keeping something reasonably large in case someone else runs into the same issue.

As an alternative, I could also put the sleep value in an envar that defaults to 0 and print a message if the first command fails that the envar may need to be set.

Reply:
Ok, I understand it better now. Let's leave the speed optimization for another time. Having it run correctly now is a great improvement.

Reply:
Sounds good.  Do you want me to open an issue for a followup and assign it to myself?